src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystem.java

```diff
@@ -227,26 +232,51 @@ private static void createBranchActivePowerTargetEquation(LfBranch branch, Discr
         }
     }
 
+    private static void createDiscreteVoltageControlEquation(LfBus bus,  VariableSet variableSet, EquationSystem equationSystem) {
+        if (bus.isDiscreteVoltageControlled()) {
+            equationSystem.createEquation(bus.getNum(), EquationType.BUS_V).addTerm(new BusVoltageEquationTerm(bus, variableSet));
+        }
+        if (bus.isDiscreteVoltageControlled() && bus.getDiscreteVoltageControl().getControllers().size() > 1) {
```

Comment:
you could insert that if in the if above. Or we could add a isDiscreteVoltageSharedControlled method?